File: three_part/part.java

	
	public static void main(String[] args)
    {
			Scanner sc=new Scanner(System.in);
            int n = sc.nextInt(); 
			int []arr= Arrays.stream(sc.next().split("[,\\[\\]]")).filter(w-> !w.equals("")).mapToInt(Integer::parseInt).toArray();
		    Solution g=new Solution();
            int []ans=g.threeEqualParts(arr,n);  
            for(int i=0;i<2;i++)
            System.out.print(ans[i] + " ");  
          
	}
}
class Solution
{
	public static int[] threeEqualParts(int arr[],int n)
        {
            int[] binary_value = new int[2];
			//Write your code here without removing the existing code
	        //the variable 'arr' contains array of integers of size n and an integer k.
	        //modified the array 'binary_value' having the subarray with k sum.
			return binary_value;
        }
}

File: three_part/part.js
var stdin= process.openStdin();
stdin.addListener("data", function(d)
{
var x= d.toString().trim().split("\n");
let b= Number(x[0]);
let sub = JSON.parse(x[1]);
var res= threeEqualParts(sub,b);
let s='';
for(var i=0;i<2;i++)
{
s+=res[i]+" ";
}
console.log(s);
});



function threeEqualParts(arr,n)
{
	var binary_value=[0,0];
    //Write your code here without removing the existing code
	//the variable 'arr' contains array of integers of size n.
	//modified the array 'binary_value' return the output of the program.
	return binary_value;
    
}

